src/DeepHelloWorld/Tests/Frontend/angular/controllers/SayHelloController.spec.js

```diff
@@ -16,11 +17,44 @@ describe('Controllers', function() {
   }));
 
   describe('SayHelloController', function() {
+    let controllerArgs = {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).